jenkins.sh

```diff
@@ -0,0 +1,4 @@
+virtualenv venv
+. venv/bin/activate
+pip2.7 install pytest toil boto s3am
```

Comment:
You shouldn't mix multiple applications in a single virtualenv. The reason we use virtualenv's is to isolate applications and their dependencies from each other. In fact, the most recent build error

http://jenkins.cgcloud.info/job/toil-scripts/11/testReport/test/test_urls/test_upload_and_download_with_encryption/

is a result of mixing s3am and Toil in a venv. They each depend on a different version of bd2k-python-lib and one wins. In this case the older one from Toil wins and that breaks s3am.